### ISSUE TITLE
acomodar el styling del detalle de capitulos

### ISSUE THREAD

**Issue by SKOLZ** [frontend-army/landing#4]

**Comment by pabloferro** [comment]
@SKOLZ que cosas te gustaria acomodar?

Tiene que ver con #29 ?

**Comment by SKOLZ** [comment]
@pabloferro no, esto es una vista especial para cada capitulo que habia creado frank, creo uqe en el stream vos la viste y no sabias que era y viste que no se podia acceder o que la codeamos y no estaba en uso. bueno si, la codeamos un poco pero no la terminamos porque queriamos ir live entonces quedo oculto. basicamente seria desempolvar esa vista y ajustar la ui para que se vea tan fachera como el resto de la pagina
